README.md

```diff
@@ -2,7 +2,7 @@
 
 # Mobile Version
 
-![Screenshot](/Images/Screenshot.png)
+![Screenshot](/Images/Screenshot3.png)
 
 > This project is a Microverse project given to students and it entails replicating a figma mobile version template
```

Comment:
- [ ] [OPTIONAL] Your description is good, but it can be better, try to explain what you did, what kind of technologies or techniques you used, what is the main scope, and so on.
- [ ] [OPTIONAL] You may fix the size for the screenshot due to is too big, 😅

Reply:
Please how can I fix the size of the screenshot in my README.md file?

Reply:
Sure! You can use `img` tag instead of using `![Screenshot](PATH_IMAGE)`
For your particular case, you can try: 
```
<img src="PATH_IMAGE" height="100px">
```
You can modify 100px for any value that you consider good :)

Reply:
I have adjusted the size of the screenshot in the README.,md file to 200*1000. I also edited my description for the pull request.

Thank you.